### PR TITLE
Feat: Add Github Issue Bot Template for Node.js (TypeScript)

### LINKED ISSUE #147: 🚀 Feature: Add Github Issue Bot for Node.js (TypeScript)
Appwrite Functions allow you to extend and customize your Appwrite server functionality by executing your custom code. 🤩 You can learn more at our official [Appwrite Functions docs](https://appwrite.io/docs/functions).

Your task is to implement the Github Issue Bot function in Node.js (TypeScript). You can look at the existing [Appwrite Functions Templates](https://github.com/appwrite/templates) in another coding language.

Your function should behave and be implemented similarly to existing Github Issue Bot Node.js template.

### Tasks summary:

- [ ] Fork & clone Appwrite's [templates](https://github.com/appwrite/templates) repository
- [ ] Create a branch `feat-implement-github-issue-bot-node-typescript`
- [ ] Write code for the function in folder `node-typescript/github-issue-bot/`. 
- [ ] Test the function using Appwrite, **provide a video**.
- [ ] Write a `README.md` explaining what the function does, how to use it, and customize it. Here is [an example](https://github.com/appwrite/templates/tree/main/node/github-issue-bot).
- [ ] Submit a pull request to our [templates](https://github.com/appwrite/templates) repository

If you need any help, contact us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

### PR DESCRIPTION
**Pull request by vimode**
## What does this PR do?

Adds the Github Issue Bot Template for Node.js(Typescript).

## Test Plan

200 OK
---
https://github.com/appwrite/templates/assets/39148877/482e6c56-3fb2-486a-96c6-862091b26abf

## Related PRs and Issues

Closes #147 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by Haimantika — COMMENTED** [review]
LGTM

**Review by loks0n — APPROVED** [review]

File: node-typescript/github-issue-bot/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* Throws an error if any of the keys are missing from the object. */
+
```
Comment (loks0n) [inline node-typescript/github-issue-bot/src/utils.ts:2]: Can we remove the space and use the docstring comment format?
e.g.

/*
 * Blah bla blah ...
 */

**Review by loks0n — CHANGES_REQUESTED** [review]

**Comment by loks0n** [comment]
Great PR! One small nitpick

**Comment by vimode** [comment]
@loks0n Thank you. 

Updated the code comment as  per request.

**Review by loks0n — APPROVED** [review]
Amazing work! Merged

**Comment by gewenyu99** [comment]
Hey there! There were a lot of big PRs during this Hacktoberfest, and we wanted to give everyone ample time to collaborate with our engineering team. If you were able to merge your PRs during October, amazing. If it’s still not merged, don’t worry about it either. Either way, we’ve got your Hacktoberfest swag minted and ready to ship. 

Please comment with your Discord username here so we can contact you about your shipping information to deliver your Hacktoberfest swag.

**Comment by vimode** [comment]
Hey @gewenyu99 my discord is vimiaou :)

**Comment by gewenyu99** [comment]
Be in touch soon!

**Pull request merged**